Version 3.0 Release Candidate (#159)
nf-core/cutandrun: docs/usage.md

There are some options detailed on the parameters page that are prefixed with `save`, `skip` or `only`. These are flow control options that allow for saving additional output to the results directory, skipping unwanted portions of the pipeline or running the pipeline up to a certain point, which can be useful for testing.

### Genome Configuration

The easiest way to run the pipeline is by using one of the pre-configured genomes that reflect the available genomes at [iGenomes]([AWS iGenomes](https://ewels.github.io/AWS-iGenomes/)). Assign `genome` to one of the key words for iGenomes and all the available reference data will be automatically fetched. The pipeline uses the following reference data:

- Target genome FASTA

- Target genome Bowtie2 Index

- Target genome GTF

- Target genome BED (will be generated from the GTF if not supplied)

- Target genome Blacklist (blacklist files for major genomes are included in the pipeline)

- Spike-in genome FASTA

- Spike-in genome Bowtie2 Index

> If the `genome` parameter is not supplied, the user must provide all the target genome data themselves (except the gene BED file). The spike-in genome defaults to e.coli and usually is not changed.

Reviewer: not sure what is being said here 'spike-in genome defaults to e.coli and usually is not changed.'

Author: Yeah I will reword - its saying the default genome is ecoli for spike-in

### Read Filtering and Duplication

> After alignment using Bowtie2 reads are filtered for mapped reads that pass a minimum quality threshold. This can be changed using the `minimum_alignment_q_score` parameter.

Reviewer: ```suggestion
After alignment using Bowtie2, mapped reads are filtered to remove those which do not pass a minimum quality threshold. This threshold can be changed using the `minimum_alignment_q_score` parameter.
```

Author: fixed
